tests/test_loaders_velociraptor.py

```diff
@@ -8,61 +9,36 @@
 from ._utils import absolute_path, mkdirs
 
 
+def create_paths(sub_dir: str) -> list[str]:
+    return [
+        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/",
+        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/$Extend",
+        f"uploads/{sub_dir}/%5C%5C.%5CC%3A/windows/system32",
+        f"uploads/{sub_dir}/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
+    ]
+
+
 @pytest.mark.parametrize(
-    "paths",
-    [
-        (
-            [
-                "uploads.json",
-                "uploads/mft/%5C%5C.%5CC%3A/",
-                "uploads/mft/%5C%5C.%5CC%3A/$Extend",
-                "uploads/mft/%5C%5C.%5CC%3A/windows/system32",
-                "uploads/mft/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
-            ]
-        ),
-        (
-            [
-                "uploads.json",
-                "uploads/ntfs/%5C%5C.%5CC%3A/",
-                "uploads/ntfs/%5C%5C.%5CC%3A/$Extend",
-                "uploads/ntfs/%5C%5C.%5CC%3A/windows/system32",
-                "uploads/ntfs/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
-            ]
-        ),
-        (
-            [
-                "uploads.json",
-                "uploads/ntfs_vss/%5C%5C.%5CC%3A/",
-                "uploads/ntfs_vss/%5C%5C.%5CC%3A/$Extend",
-                "uploads/ntfs_vss/%5C%5C.%5CC%3A/windows/system32",
-                "uploads/ntfs_vss/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
-            ]
-        ),
-        (
-            [
-                "uploads.json",
-                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/",
-                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/$Extend",
-                "uploads/lazy_ntfs/%5C%5C.%5CC%3A/windows/system32",
-                "uploads/lazy_ntfs/%5C%5C%3F%5CGLOBALROOT%5CDevice%5CHarddiskVolumeShadowCopy1",
-            ]
-        ),
-    ],
+    "sub_dir",
+    ["mft", "ntfs", "ntfs_vss", "lazy_ntfs"],
```

Comment:
Since I see that "auto" has been added for Windows, it would be nice to add it to the test as well.

```suggestion
    ["mft", "ntfs", "ntfs_vss", "lazy_ntfs", "auto"],
```